README.md

```diff
@@ -121,7 +121,7 @@ Examples: see [example/angular1](https://github.com/google/dart-scissors/tree/ma
   ```
 
 Valid settings:
-- `pruneCss` (boolean): by default, `true` in `release` only
+- `pruneCss` (`skip` or `print` or `overwrite`): by default, `overwrite` in `release` only
```

Comment:
As discussed, can you deprecate the existing flag and add a new one? I found 1 existing usage of pruneCss with a bool value, this would break it.

Alternatively, we can keep it and accept `true`, `false`, `warnOnly`